spec/models/round_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Round do
+  it 'should have a valid factory' do
+    k = FactoryBot.create(:round, :fall)
+    k.start_date = Time.new(k.session.year, 9)
```

Comment:
All start_dates should be required to have a date. I think we should add validation on the models to enforce this. Doesn't make sense for a round or session to be able to start on entire month.